ndarray/ndarray-tfjs/src/jsMain/kotlin/io.kinference.ndarray/extensions/NDArrayExtension.kt

```diff
@@ -441,3 +443,48 @@ suspend fun NumberNDArrayTFJS.fmod(other: NumberNDArrayTFJS): NumberNDArrayTFJS
 suspend operator fun NumberNDArrayTFJS.rem(other: NumberNDArrayTFJS) = fmod(other)
 
 suspend fun NumberNDArrayTFJS.pow(exp: NumberNDArrayTFJS) = NumberNDArrayTFJS(tfjsArray.pow(exp.tfjsArray))
+
+suspend fun oneHot(indices: NumberNDArrayTFJS, depth: Int, values: NDArrayTFJS, axis: Int = -1): NDArrayTFJS {
```

Comment:
Maybe also move it to `NDArrayTFJS.Companion`? BTW we can infer types here, I think it might be useful